experiments/scripts/conversation.py

```diff
@@ -0,0 +1,81 @@
+"""Script for ."""
+import json
+from datetime import datetime
+import os
+from pathlib import Path
+
+import configparser
+import click
+import yaml
+
+from src.ai_helps_pwr.utils.common import (
+    load_model_from_config,
+    load_prompt
+)
+
+
+@click.command()
+@click.option(
+    "--config_path",
+    help="Path to config.",
+    type=click.Path(exists=True, path_type=Path),
+    default=Path("config/config.local")
+)
+@click.option(
+    "--output_dir",
+    help="Directory to save data.",
+    type=click.Path(path_type=Path),
+    default=Path("output")
+)
+@click.option(
+    "--hparams_path",
+    help="Path to selected model hparams",
+    type=click.Path(exists=True, path_type=Path),
+    default=Path("experiments/config/models.yaml")
+)
+@click.option(
+    "--model",
+    help="Name of selected  model",
+    type=str,
+    default="qpt_conversation"
+)
+@click.option(
+    "--output_dir",
+    help="Directory to save data.",
+    type=click.Path(path_type=Path),
+    default=Path("output")
+)
+def main(
+    config_path: Path,
+    hparams_path: Path,
+    model: str,
+    output_dir: Path
+):
+    """Use GPT to answer a question."""
+    config = configparser.RawConfigParser()
+    config.read(config_path)
+
+    gpt_config = dict(config.items('gpt'))
+
+    with open(hparams_path, "r") as fin:
+        hparams = yaml.safe_load(fin)[model]
+
+    model = load_model_from_config(cfg=hparams["model"])
```

Comment:
To trochę skomplikowane, bo na razie nie wiemy, czy będziemy używać innych klas modeli, więc by można było po prostu:
```
gpt_model = GPT(**hparams["model"]["kwargs"])
```

Reply:
Wierzę, że możemy mieć inne modele jak t5 itd